WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -282,11 +282,17 @@ class RegisterDomainSuggestionsViewController: UIViewController {
 // MARK: - DomainSuggestionsTableViewControllerDelegate
 
 extension RegisterDomainSuggestionsViewController: DomainSuggestionsTableViewControllerDelegate {
-    func domainSelected(_ domain: FullyQuotedDomainSuggestion) {
-        WPAnalytics.track(.automatedTransferCustomDomainSuggestionSelected)
-        coordinator?.domain = domain
-        showButton(animated: true)
-        hideTransferFooterView(animated: true)
+    func domainSelected(_ domain: FullyQuotedDomainSuggestion?) {
+        self.coordinator?.domain = domain
+
+        if domain != nil {
+            WPAnalytics.track(.automatedTransferCustomDomainSuggestionSelected)
+            showButton(animated: true)
+            hideTransferFooterView(animated: true)
+        } else {
+            hideButton(animated: true)
+            showTransferFooterView(animated: true)
```

Comment:
[Nit] What do you say we extract this to a helper function, to call it here and in `newSearchStarted()`?

Reply:
Agreed, I'll open a new PR following up on this and the mysterious separator. 🙏